scss/_base_blockquotes.scss

```diff
@@ -4,6 +4,12 @@
 @mixin vf-b-blockquotes {
   blockquote {
     border-left: 2px solid $color-mid-dark;
+    margin-bottom: 1.1rem;
+    padding-bottom: 0.4rem;
```

Comment:
I think we should avoid magic numbers where possible. Is this derived from paragraph spacing?